### DISCUSSION TITLE
Attribute bindings are being converted to lowercase

### DISCUSSION THREAD

**Discussion by Fill84** [livewire/livewire#8890] [7. Bugs (archived)] [answered]
### Livewire version

v3.5.8

### Laravel version

v11.25.0

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

1: Set up an SVG with an attribute using x-bind in Livewire: In a Livewire component, bind the viewBox attribute to a variable using x-bind:viewBox="variableName".

2: Inspect the rendered HTML: After the component is rendered, inspect the SVG element in the browser's developer tools.

3: Observe the incorrect output: Notice that the viewBox attribute has been converted to viewbox, resulting in incorrect rendering or behavior of the SVG element.

### Repository or Wirebox URL

none

### Additional information

the x-bind attribute converts the attribute to lowercase, which is not ideal as i said in the steps to reproduce.

`<svg xmlns="http://www.w3.org/2000/svg" width="21px" height="21px"  :viewBox="quality.viewBox" fill="currentColor">
    <path :d="quality.path" />
</svg>`

this will become

`<svg xmlns="http://www.w3.org/2000/svg" width="21px" height="21px"  :viewBox="quality.viewBox"  viewbox="..." fill="currentColor">
    <path :d="quality.path"  d="..."  />
</svg>`

which renders the svg incorrect

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by EdgarsJoja** [comment, marked as answer]
Try
```blade
:view-box.camel="..."
```

it seems it's an undocumented `x-bind` modifier in Alpine. See https://github.com/alpinejs/alpine/issues/594

**Reply by Fill84** [reply]
This is working, thanks
